climada/hazard/test/test_base_xarray.py

```diff
@@ -342,9 +342,7 @@ def test_missing_dims(self):
             ds = ds.isel(time=0).squeeze()
             hazard = Hazard.from_xarray_raster(ds, "", "")
             self._assert_default_types(hazard)
-            np.testing.assert_array_equal(
-                hazard.event_name, [np.datetime64(self.time[0])]
-            )
```

Comment:
Why remove the test instead of adapting it to the new expected values? 🤔

Reply:
To my understanding this test (test_missing_dims) is supposed to check whether everything still works after removing time as a dimension. Since the default hazard.event_name used to depend on the time it was necessary to test whether this still returned the correct default event_name. My thinking was that if event_name is not connected to time anymore it would not be necessary to test the default name in this specific test

Reply:
Yes, I agree. Thanks for clarifying my own test 🙈

Reply:
Anytime :D Now I will keep it in and adapt it based on the discussion above though